app/services/proxy_status.rb

```diff
@@ -26,10 +26,24 @@ def tftp_server
     end
   end
 
+  # TODO: extract to another status implementation
```

Comment:
It looks like status is going to be merged today, so maybe this todo has to be completed